src/modules/home/routes/HomeRoute.js

```diff
@@ -0,0 +1,6 @@
+import { Home } from "../components/Home";
+
+export const homeRoute = {
+	path: "/",
+	element: <Home />,
+};
```

Comment:
One thing I shouldve mentioned Id like is path to be saved into a const. I think later we will use it for navigation

const homePath = '/'

export const homeRoute = {
	path: "homePath",
	element: <Home />,
};

Reply:
So in future if we wanna route there from a function we can export it and route